### PR TITLE
Update optioncodes.md

### PR DESCRIPTION
**Pull request by lyceel**
These were primarily gathered by trawling through lots of various Tesla forums. I don't make any guarantee as to their accuracy, but it's the best I've been able to put together so far.

### CODE REVIEW

**Comment by scootklein** [comment]
bless you for doing this 🙌

**Comment by timdorr** [comment]
Thanks!

File: docs/vehicle/optioncodes.md [hunk 1]
```diff
@@ -1,68 +1,253 @@
...
+| LT01 | Standard interior | | 
+| LT1B | LT1B | | 
+| LT3W | LT3W | | 
+| LT4B | LT4B | | 
+| LT4C | LT4C | | 
+| LT4W | LT4W | | 
+| LT5C | LT5C | | 
+| LT5P | LT5P | | 
+| ME02 | Memory Seats | | 
+| MI00 | 2015 Production Refresh | |
```
Comment (jab416171) [inline docs/vehicle/optioncodes.md]: This can't be right, my 2018 Model 3 has this option code.

**Comment by lyceel** [comment]
Model 3 option codes can't be trusted. Last time I looked, every Model 3
returned the same option codes (your paint code is probably wrong, too).

For Model 3, the vehicle configuration API is the only reliable source.

**Comment by jab416171** [comment]
Yup, well that's awkward. The paint code is PBSB, but my car's definitely blue. In the same API response: "exterior_color": "DeepBlue"

**Pull request merged**